mypy.ini

```
@@ -0,0 +1,18 @@
+[mypy]
+files =
+  docs,
+  share,
+  src,
+  tests
+# FIXME: get rid of missing imports ignore
+ignore_missing_imports = true
```

Just to be clear about https://github.com/ansible/ansible-navigator/pull/979/files#r810629960, this is what I expect:
```suggestion
ignore_missing_imports = true  # FIXME: get rid of missing imports ignore
```
(unless it breaks the parser, of course)

---

It doesn't break the parse, but the value ends up not being a boolean, which is why it is on the preceding line.

```
mypy.ini: [mypy]: ignore_missing_imports: Not a boolean: true  # FIXME: get rid of missing imports ignore
```

---

BTW, I expect missing imports to be ignored.

---

> It doesn't break the parser, but the value ends up not being a boolean, which is why it is on the preceding line.

Oh, okay. That's what I was afraid of. I new that `configparser` would have some of such quirks and sometimes the applications (one example where I fixed it is `tox`) would cut the comments off but sometimes this is not implemented. It's disappointing that it's not possible but at least the "why" is now documented here. Resolving.